Formula/cassandra.rb

```diff
@@ -13,25 +13,17 @@ class Cassandra < Formula
   end
 
   depends_on "cython" => :build
-  # Due to Python 2 (https://issues.apache.org/jira/browse/CASSANDRA-10190), cassandra 4 will support python3
-  depends_on :macos
-  depends_on "openjdk@8" # cassandra 4 will support Java 11
+  depends_on "python@3.9" => :build
+  depends_on "openjdk@11"
 
-  # Only >=Yosemite has new enough setuptools for successful compile of the below deps.
-  # Python 2 needs setuptools < 45.0.0 (https://github.com/pypa/setuptools/issues/2094)
   resource "setuptools" do
-    url "https://files.pythonhosted.org/packages/b2/40/4e00501c204b457f10fe410da0c97537214b2265247bc9a5bc6edd55b9e4/setuptools-44.1.1.zip"
-    sha256 "c67aa55db532a0dadc4d2e20ba9961cbd3ccc84d544e9029699822542b5a476b"
-  end
-
-  resource "futures" do
-    url "https://files.pythonhosted.org/packages/47/04/5fc6c74ad114032cd2c544c575bffc17582295e9cd6a851d6026ab4b2c00/futures-3.3.0.tar.gz"
-    sha256 "7e033af76a5e35f58e56da7a91e687706faf4e7bdfb2cbc3f2cca6b9bcda9794"
+    url "https://files.pythonhosted.org/packages/db/e2/c0ced9ccffb61432305665c22842ea120c0f649eec47ecf2a45c596707c4/setuptools-57.4.0.tar.gz"
+    sha256 "6bac238ffdf24e8806c61440e755192470352850f3419a52f26ffe0a1a64f465"
   end
 
   resource "six" do
-    url "https://files.pythonhosted.org/packages/6b/34/415834bfdafca3c5f451532e8a8d9ba89a21c9743a0c59fbd0205c7f9426/six-1.15.0.tar.gz"
-    sha256 "30639c035cdb23534cd4aa2dd52c3bf48f06e5f4a941509c8bafd8ce11080259"
+    url "https://files.pythonhosted.org/packages/71/39/171f1c67cd00715f190ba0b100d606d440a28c93c7714febeca8b79af85e/six-1.16.0.tar.gz"
+    sha256 "1e61c37477a1626458e36f7b1d82aa5c9b094fa4802892072e49de9c60c4c926"
   end
```

Comment:
Let's do `depends_on "six"` instead.

Reply:
thanks for the help! i may have missed something, but it doesn't seem like `six` is installed into the venv this way?

Reply:
You're right that it's not, but `venv`s that `brew` creates [have access to system site-packages](https://github.com/Homebrew/brew/blob/3e52e8b48ab036af77a582219ea250107c84432f/Library/Homebrew/language/python.rb#L221).

Reply:
ah i see now